pylint/checkers/exceptions.py

```diff
@@ -555,7 +566,9 @@ def visit_tryexcept(self, node: nodes.TryExcept) -> None:
                         and not _is_raising(handler.body)
                     ):
                         self.add_message(
-                            "broad-except", args=exception.name, node=handler.type
+                            "broad-exception-caught",
+                            args=exception.name,
+                            node=handler.type,
```

Comment:
```suggestion
                            node=handler.type,
                            confidence=interface.HIGH,
```
AFAIK.

Reply:
Correct. Have this in a commit not pushed. I made a draft PR to see how the primer reacted. Not ready for review yet.